src/state/store.js

```diff
@@ -5,21 +5,29 @@ import { createStore } from '../utils/create-store';
 // FIXME: take care of this dependency cycle.
 import { probeSet } from './telemetry-search'; // eslint-disable-line import/no-cycle
 
-import CONFIG from '../config/firefox-desktop';
+import productConfig from '../config/products';
+
+function getProduct() {
+  const defaultProduct = 'firefoxDesktop';
+  const mappings = {
```

Comment:
this mapping should probably live in `../config/products`, not here. That's the point of the config file.

Reply:
Also, we should remove the _need_ for this mapping. Are we not coding things properly b/t the API & GLAM? This should be a goal imo. Might be worth getting @robhudson's input here. If we have to ahve this mapping, that's fine, but would be worth filing an issue so we don't have to maintain this.

Reply:
Furthermore, we could have `getProduct` be a function in `productConfig` I think.